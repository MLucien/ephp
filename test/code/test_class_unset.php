<?php

class Simple {

    public function hello() {
        print "Hello world!\n";
    }

}

$s1 = new Simple();
$s2 = new Simple();
var_dump($s1);
var_dump($s2);
unset($s1);
var_dump($s1);
var_dump($s2);

