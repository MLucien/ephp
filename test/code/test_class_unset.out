object(Simple)#1 (0) {
}
object(Simple)#2 (0) {
}
NULL
object(Simple)#2 (0) {
}
